modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/paths/WorkflowMetricsProcessListView.path

```diff
@@ -0,0 +1,29 @@
+<html>
+<head>
+<title>WorkflowMetricsProcessListView</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">WorkflowMetricsProcessListView</td></tr>
+</thead>
+<tbody>
+<tr>
+	<td>PAGINATION_NEXT_BUTTON</td>
+	<td>//ul[contains(@class,'pagination')]/li/a[contains(.,'Next')] | //ul[contains(@class,'pagination')]/li[@class='']/a/span[contains(@class,'icon-caret-right')] | //button[contains(@class, 'page-link') and contains(@aria-label,'Next')]</td>
+	<td></td>
```

Comment:
Please, change it to:
`<td>//button[contains(@class, 'page-link') and contains(@aria-label,'Next')]</td>`